pint/compat.py

```diff
@@ -148,7 +148,7 @@ def _to_magnitude(value, force_ndarray=False):
 
 def is_upcast_type(other):
     # Check if class name is in preset list
```

Comment:
Would you be able to add documentation emphasizing that `other` is expected to be a type object?

Reply:
what do you think of
```
check if the type object is a upcast type

:param other: type
```

Reply:
That sounds good for now!

Reply:
done